cmd/generate/action/template/cmd/execute/runner.go

```diff
@@ -62,6 +62,16 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 		if err != nil {
 			return microerror.Mask(err)
 		}
+
+		err = clients.InitControlPlane(ctx)
+		if err != nil {
+			return microerror.Mask(err)
+		}{{ if ne .Action "ac001" }}
+
+		err = clients.InitTenantCluster(ctx)
+		if err != nil {
+			return microerror.Mask(err)
+		}{{ end }}
```

Comment:
This is super ugly but I could not make it work in a reasonable amount of time otherwise. The normal user should never have to bother with the template anyway and the generated code is properly formatted.